static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -97,19 +112,25 @@ const initForms = (): void => {
     }
 
     // insert hidden extra data into forms
-    $.forEachElement('#feedback__form input[name=extra]', elem => {
-        elem.value = JSON.stringify(getExtraDataInformation(), null, '  ');
+
+    return getExtraDataInformation().then(extraData => {
+        console.log(extraData);
```

Comment:
Do we still want to `console.log(extraData)` here? 
Although maybe it is useful for debugging. 🤔😅